### ISSUE TITLE
Installation requires MySQL Password

### ISSUE THREAD

**Issue by ghost** [osTicket/osTicket#5105]
### Prerequisites

* [ Yes ] Can you reproduce the problem in a fresh installation of the "develop" branch?
* [ No ] Do you have any errors in the PHP error log, or javascript console?
* [ Yes ] Did you check the [osTicket forums](https://forum.osticket.com/)?
* [ Yes ] Did you [perform a cursory search](https://github.com/osTicket/osTicket/issues) to see if your bug or enhancement is already reported?

For more information on how to write a good [bug report](https://github.com/osTicket/osTicket/wiki/Github-Issues-Guidelines)

### Description

[Description of the bug or feature]

osTicket doesn't allow installation with a blank MySQL password.

I have no MySQL password because of local testing and this is the only script I have used that forces me to insert a password.

### Steps to Reproduce

1. [First Step]
Visit installation page of a fresh install
2. [Second Step]
Type in details and press install now
3. [and so on...]
You're screwed

**Expected behavior:** [What you expected to happen]

Accept an empty MySQL password

**Actual behavior:** [What actually happened]

" Missing or invalid data — Correct any errors below and try again "

### Versions

1.12.3

**Comment by JediKev** [comment]
@groovybluedog

Yes, we require a database password to install osTicket as in pretty much every case you have a password set for database user accounts to secure the database(s). (especially since the database houses customer data) The only case you don’t set a password for database user accounts is maybe in a dev environment or something.

With this being said, we will take this as feedback.

Cheers.
